main.js

```diff
@@ -0,0 +1,286 @@
+    playerTurn.style.display = "none";
+    let x_turn = true;
+    let roundCount = 0;
+    let player1name;
+    let player2name;
+    const winningCombinations = [
+        [0, 1, 2],
+        [3, 4, 5],
+        [6, 7, 8],
+        [0, 3, 6],
+        [1, 4, 7],
+        [2, 5, 8],
+        [0, 4, 8],
+        [2, 4, 6],
+    ];
+
+    const cellElements = document.querySelectorAll('[data-cell]');
+    let cellArray = Array.from(cellElements);
+
+
+
+    // create module to control display
+    const displayController = (() => {
+
+        // select for "home screen" buttons
+        const restartBtn = document.getElementById('restart-btn');
+        restartBtn.addEventListener('click', resetDisplay);
+        restartBtn.style.display = 'none';
+        restartBtn.addEventListener('click', resetDisplay);
+            function resetDisplay () {
+                playerTurn.innerText = '';
+                playerTurn.style.display = 'none';
+                playerOneNameInput.style.display = 'inline';
+                playerTwoNameInput.style.display = 'inline';
+                startGame.style.display = 'inline';
+                startBtn.style.display = 'inline';
+                restartBtn.style.display = 'none';
+                cellElements.forEach(cell => {
+                    cell.innerText = '';
+                    cell.setAttribute('id', '');
+                    cell.setAttribute('class', 'cell');
+                });
+            }
+
+        const startGame = document.forms['start-game-form'];
+        startGame.addEventListener('submit', startFunction);
+        const playerOneNameInput = startGame.querySelector('[data-player-1-input]')
+        const playerTwoNameInput = startGame.querySelector('[data-player-2-input]')
+
+
+        const startBtn = document.getElementById('start-game-btn');
+    
+        // create function to hide home screen and start game
+        function startFunction(e) {
+            e.preventDefault();
+            activateCells();
+            let playerOneNameInputVal = startGame.querySelector('[data-player-1-input]').value;
+            let playerTwoNameInputVal = startGame.querySelector('[data-player-2-input]').value;
+            player1name = playerOneNameInputVal;
+            player2name = playerTwoNameInputVal;
+       
+            playerOneNameInput.style.display = 'none';
+            playerTwoNameInput.style.display = 'none';
+            startGame.style.display = 'none';
+            startBtn.style.display = 'none';
+            playerTurn.innerText =`${playerOneNameInputVal}'s turn`;
+            playerTurn.style.display = 'inline';
+        };
+
+
+        return {
+            restartBtn,
+            startFunction,
+            resetDisplay,
+        }
+    })();
+
+    // create array from cells in order to grab index # of cell
+    function activateCells () {
+        cellElements.forEach(cell => {
+            cell.addEventListener('click', addToBoard, { once: true });
```

Comment:
This is a great way to do this! Adding the `once` attribute on it is a great way to stop people from clicking in the same cell again